core/src/main/java/com/ritense/valtimo/service/CamundaTaskService.java

```diff
@@ -78,6 +74,19 @@
     private final RuntimeService runtimeService;
     private final UserManagementService userManagementService;
 
+    public CamundaTaskService(TaskService taskService, FormService formService, ContextService contextService, DelegateTaskHelper delegateTaskHelper, CamundaTaskRepository camundaTaskRepository, CamundaProcessService camundaProcessService, Optional<ResourceService> optionalResourceService, ApplicationEventPublisher applicationEventPublisher, RuntimeService runtimeService, UserManagementService userManagementService) {
```

Comment:
This long line isn't easily readable. Maybe move them to a line per parameter?

There is a LineLength of max 160 characters in the checkstyle document. Does that not trigger here?